src/preface.md

```diff
@@ -98,9 +98,31 @@ $ # QEMU
 $ sudo apt-get install qemu-system-arm
 ```
 
+#### Ubuntu 18.04
+
+``` console
+$ # gdb-multiarch -- use `gdb-multiarch` when you wish to invoke gdb
+$ sudo apt-get install gdb-multiarch
+
+$ # QEMU
+$ sudo apt-get install qemu-system-arm
+```
+
 #### Windows
 
 - [arm-none-eabi-gdb](https://developer.arm.com/open-source/gnu-toolchain/gnu-rm/downloads).
   The GNU Arm Embedded Toolchain includes GDB.
 
 - [QEMU](https://www.qemu.org/download/#windows)
+
+## Installing a toolchain bundle from ARM (optional step) (tested on Ubuntu 18.04)
+- With the late 2018 switch from
+[GCC's linker to LLB](https://rust-embedded.github.io/blog/2018-08-2x-psa-cortex-m-breakage/) for Cortex-M microcontrollers, [gcc-arm-none-eabi][1] is no longer 
```

Comment:
```suggestion
[GCCs linker to LLD](https://rust-embedded.github.io/blog/2018-08-2x-psa-cortex-m-breakage/) for Cortex-M microcontroller, [gcc-arm-none-eabi][1] is no longer 
```

Reply:
I also think you'll need to move the link out of the verbatim section to make it work.

Reply:
Ah!  I thought that was my local renderer!  Thanks for the catch.